bin/install-package-tests

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
```

Comment:
I don't think all these scripts here in `bin/` are needed. The `wp-cli/wp-cli-tests` package already provides those. Hence you can use the composer scripts `composer prepare-tests` etc.